src/emcpy/plots/create_map.py

```diff
@@ -0,0 +1,294 @@
+            except KeyError:
+                raise TypeError(f'{plotobj} is not a valid plot type.' +
+                                'Current plot types supported are:\n' +
+                                f'{" | ".join(feature_dict.keys())}"')
+
+    def _scatter(self, plot):
+        """
+        Plots MapScatter object on map axes.
+        """
+
+        cs = self.ax.scatter(plot.longitude,
+                             plot.latitude,
+                             c=plot.data,
+                             s=plot.markersize,
+                             cmap=plot.cmap,
+                             marker=plot.marker,
+                             vmin=plot.vmin,
+                             vmax=plot.vmax,
+                             transform=self.projection.projection)
+        if plot.colorbar:
+            self.cs = cs
+
+    def _gridded(self, plot):
+        """
+        Plots MapGridded object on map axes.
+        """
+
+        cs = self.ax.pcolormesh(plot.longitude,
+                                plot.latitude,
+                                plot.data,
+                                cmap=plot.cmap,
+                                vmin=plot.vmin,
+                                vmax=plot.vmax,
+                                alpha=plot.alpha,
+                                transform=self.projection.projection)
+
+        if plot.colorbar:
+            self.cs = cs
+
+    def _contour(self, plot):
+        """
+        Plots MapContour object on map axes.
+        """
+
+        cs = self.ax.contour(plot.longitude,
+                             plot.latitude,
+                             plot.data,
+                             levels=plot.levels,
+                             colors=plot.colors,
+                             linewidth=plot.linewidths,
+                             linestyle=plot.linestyles,
+                             cmap=plot.cmap,
+                             vmin=plot.vmin,
+                             vmax=plot.vmax,
+                             alpha=plot.alpha,
+                             transform=self.projection.projection)
+
+        if plot.clabel:
+            plt.clabel(cs, levels=plot.levels, use_clabeltext=True)
+
+        if plot.colorbar:
+            self.cs = cs
+
+    def add_title(self, label,
+                  loc='center',
+                  fontsize=12,
+                  fontweight='normal',
+                  color='k',
+                  va='baseline'):
+        """
+        Plots title to map axes.
+
+        Args:
+            label : (str) Text to use for title
+            loc : (str; default='center') Location of title
+            fontsize : (int; default=12) Text font size
+            fontweight : (str; default='normal') Text font weight
+            color : (str; default='k') Text font color
+            va : (str; default='baseline') Vertical alignment of text
+        """
+
+        self.ax.set_title(label, loc=loc, fontsize=fontsize,
```

Comment:
Just asking, can you call this 2x and it will add two titles? Or will it overwrite the first one?

Reply:
If you call it twice without specifying the location, it will overwrite the first one. If you call it twice and specify the location as 'right', 'left', or 'center', it will plot two titles.

Example using commands:
```
mymap.add_title(label='EMCPy Map', loc='left', fontsize=14)
mymap.add_title(label='second title', loc='right', fontsize=14)
```
![image](https://user-images.githubusercontent.com/69815622/129604558-7c40c3bb-d483-4a4a-ab6c-b54958cd0234.png)

Here is what happens if you call the same location twice:
```
mymap.add_title(label='EMCPy Map', loc='center', fontsize=14)
mymap.add_title(label='second title', loc='center', fontsize=14)
```
![image](https://user-images.githubusercontent.com/69815622/129604737-83fa5a34-ccfa-4d9a-afb8-98104650c316.png)